src/ApiService/ApiService/onefuzzlib/Auth.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.OneFuzz.Service;
+
+using System.Security.Cryptography;
+
+public class Auth
+{
+    public static Authentication BuildAuth()
+    {
```

Comment:
you verified it matches Python calling ssh-key gen and it matches ?

Reply:
No i did not. I am counting on check pr to validate this